.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+    tags: '*'
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.2'
```

Comment:
Why 1.2 instead of 1.0?

Reply:
The `hasfield` function is only available on Julia 1.2 and higher. But, I figured out a way to make it work on Julia 1.0 and 1.1, so now we test on Julia 1.0 and up.